HelpSource/Classes/Method.schelp

```diff
@@ -40,14 +43,23 @@ instanceMethods::
 method::ownerClass
 returns::
 The link::Classes/Class:: for which the method is part of the implementation.
+If this is a class method, then the owner class will be a 'Meta_Class'.
```

Comment:
If the receiver is an instance method, the owner class is the class for this instance. For a class method, it is the class of the class, whose name is prefixed with "Meta_".

code::
m = Array.findRespondingMethodFor('reverse');
m.ownerClass; // Array
m = Array.class.findRespondingMethodFor(\new);
m.ownerClass; // Meta_Array
::